functorch/csrc/DynamicLayer.cpp

```diff
@@ -457,6 +462,18 @@ static DispatchKeySet keysToExcludeWhenEnteringDynamicLayer(DispatchKey key) {
   exclude = exclude - keysForEnteringDynamicLayer(key);
   return exclude;
 }
+static bool functionalAtCurrentLevel(const Tensor& tensor) {
```

Comment:
nit: isFunctionalTensorAtCurrentLevel, "functional" is an overloaded word by itself